config/initializers/ihat_supported_formats.rb

```diff
@@ -1,26 +1,2 @@
-Tahi::Application.config.ihat_supported_formats = nil
-module IhatSupportedFormats
-  def self.call
-    if ENV['IHAT_URL'].present?
-      begin
-        response = Faraday.get(ENV['IHAT_URL'])
-        if response && response.body
-          Tahi::Application.config.ihat_supported_formats =
-            JSON.dump(JSON.parse(response.body))
-        else
-          warn "Invalid JSON response from #{ENV['IHAT_URL']}"
-        end
-      rescue Faraday::ConnectionFailed
-        warn "Unable to connect to #{ENV['IHAT_URL']}"
-      end
-    else
-      warn "ENV['IHAT_URL'] Not set, falling back to default document types…"
-    end
-  end
-end
-
-def warn(message)
-  Rails.logger.warn message
-end
-
-IhatSupportedFormats.call unless Rails.env.test?
+Tahi::Application.config.ihat_supported_import_formats = %w(docx odt)
+Tahi::Application.config.ihat_supported_export_formats = %w(docx)
```

Comment:
The reason we were making the preliminary request to iHat to discover formats supported in iHat, that way we don't have to hardcode the formats this way. This array will have to be updated each time we add a format in iHat.